contracts/tests/CounterWL.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.0;
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 // solhint-disable not-rely-on-time
-contract CounterWithWhitelist is Ownable {
+contract CounterWL is Ownable {
```

Comment:
do we still need this one?

Reply:
This contract is used for testing purposes only.